docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -0,0 +1,291 @@
+in the Matter stack and user application.
+
+### Factory data components
+
+A factory data set consists of following parameters:
+
+|Key name|Full name|Length|Format|Conformance|Description|
+|:------:|:-------:|:----:|:----:|:---------:|:---------:|
+|**`sn`**|`serial number`|<1, 32> B|*ASCII string*|mandatory|A serial number parameter defines an unique number of manufactured device. Maximum length of serial number is 32 characters.|
+|**`vendor_id`**|`vendor ID`|2 B|*uint16*|mandatory|A CHIP-assigned id for the organization responsible for producing the device.|
+|**`product_id`**|`product ID`|2 B|*uint16*|mandatory|A unique id assigned by the device vendor to identify the product or device type. Defaults to a CHIP-assigned id designating a non-production or test "product".|
+|**`vendor_name`**|`vendor name`|<1, 32> B|*ASCII string*|mandatory|A human-readable vendor name which provides a simple string containing identification of device's vendor for the Content APP.|
+|**`product_name`**|`product name`|<1, 32> B|*ASCII string*|mandatory|A human-readable product name which provides a simple string containing identification of the product for the Content APP.|
+|**`date`**|`manufacturing date`|<8, 10> B|*ISO 8601*|mandatory|A manufacturing date specifies the date that the device was manufactured. The format used for providing a manufacturing date is ISO 8601 e.g. YYYY-MM-DD.|
+|**`hw_ver`**|`hardware version`|2 B|*uint16*|mandatory|A hardware version number specifies the version number of the hardware of the device. The meaning of its value, and the versioning scheme, are vendor defined.|
+|**`hw_ver_str`**|`hardware version string`|<1, 64> B|*uint16*|mandatory|A hardware version string parameter specifies the version of the hardware of the device as a more user-friendly value than that presented by the hardware version integer value. The meaning of its value, and the versioning scheme, are vendor defined.|
+|**`rd_uid`**|`rotating device ID unique ID`|<20, 36> B|*hex string*|mandatory|The rotating device ID unique ID consists of a randomly-generated 128-bit or longer octet string. This parameter should be protected against reading or writing over the air after initial introduction into the device, and stay fixed during the lifetime of the device. |
+|**`dac_cert`**|`(DAC) Device Attestation Certificate`|<1, 1204> B|*hex string*|mandatory|All commissionable Matter Nodes SHALL include a Device Attestation Certificate (DAC) and corresponding private key, unique to that Device. The DAC is used in the Device Attestation process, as part of Commissioning a Commissionee into a Fabric. The DAC is a DER-encoded X.509v3-compliant certificate as defined in RFC 5280.|
```

Comment:
@greg-fer significant part of this description is a copy-paste from the Matter spec. Are we allowed to do that, or should we cite it, as a part of external document?

Reply:
Hmm, Apart from that, it's from Matter spec, I realized that it could be incomprehensible for users in this form.